### PR TITLE
Add new typings for project "jotform-css.js"

### PR DESCRIPTION
**Pull request by PandaWood**
Please fill in this template.

- [✓ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [✓] Test the change in your own code. (Compile and run.)
- [✓ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [✓ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [✓] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [✓] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [✓] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [✓] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [✓ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [✓ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/jotform-css.js/tslint.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
```
Comment (plantain-00) [inline types/jotform-css.js/tslint.json]: This file should only `"extends": "dtslint/dt.json"` for new packages, you can disable rules in source code

**Review by plantain-00 — CHANGES_REQUESTED** [review]
This package can be used as global cssjs and `export class cssjs` https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module

File: types/jotform-css.js/tslint.json (hunk 1 above)
Comment (PandaWood) [reply types/jotform-css.js/tslint.json]: I currently get this error which makes no sense to me `In /Users/spurrymoses/code/DefinitelyTyped/types/jotform-css.js: Expected file 'jotform-css.js-global.test.ts' to be named jotform-css.js-tests.ts`

Given that I'm following instructions from further down the page, 2 have 2 different test files.
 `Name one YourLibraryName-global.test.ts and the other YourLibraryName-module.test.ts`

That instruction is at least wrong on the account that the file must actually be named `...tests` ie "with an added s" or another error will be raised.

Comment (PandaWood) [reply types/jotform-css.js/tslint.json]: Even if I add the file `jotform-css.js-tests.ts` - the error persists.

Comment (plantain-00) [reply types/jotform-css.js/tslint.json]: @andy-ms

**Comment by PandaWood** [comment]
As I mentioned in a previous comment which the bot seems to have deleted.
The current error makes no sense to me. I followed these instructions: `Name one YourLibraryName-global.test.ts and the other YourLibraryName-module.test.ts`

Those instructions are actually wrong, and the file has to be named "tests" - ie with an added "s".

But even then, following these instructions gives me this error - even though I have also included the filename it's asking for.

How/why does the big.js library still pass these tests? Is it an unsaid requirement that the files need to be in a folder named "test"?

File: types/jotform-css.js/tslint.json (hunk 1 above)
Comment (PandaWood) [reply types/jotform-css.js/tslint.json]: Sorry, I worked out I needed to add the folder "tests", I've deleted my comments to avoid clutter

**Review by plantain-00 — APPROVED** [review]

**Pull request merged**